src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -525,6 +629,30 @@ const addKeyToCollectionItems = (list) => {
     });
 }
 
+/*******************************************************************************
+ * @description Dispatches an Application Event
+ * @param eventName
+ * @param messageHeader
+ * @param messageBody
+ * @param eventSource
+ */
+const dispatchApplicationEvent = (eventName, messageHeader, messageBody, eventSource) => {
```

Comment:
I would suggest not exporting a function that uses pubsub here.  If we need to use pubsub we should fire events directly from the components but not using pubsub at all is preferrable.